CONTRIBUTING.md

```diff
@@ -38,7 +38,7 @@ Fork reek, then clone it, make sure you have
 and make sure all of the existing tests pass:
 
 ```
-git clone git@github.com:…username…/reek.git
+git clone git@github.com:troessner/reek.git
```

Comment:
No, that should be ...username..., since contributors should clone their own fork.